### DISCUSSION TITLE
Support specifying the theme with a string to support JSON and YAML configuration files

### DISCUSSION THREAD

**Discussion by pklaschka** [fliegwerk/fliegdoc#9] [Ideas]
As requested by @fussel178:

`require` is not possible in `.yml` and similar files.
Can you use a string like `fliegdoc-html-parser` instead and require it in the configuration builder?
You can also disable support for `.yml` and similar files.
See: https://www.npmjs.com/package/cosmiconfig#searchplaces

_Originally posted by @fussel178 in https://github.com/fliegwerk/fliegdoc/pull/8#discussion_r554155394_

**Comment by fussel178** [comment]
Okay, so you need in https://github.com/fliegwerk/fliegdoc/blob/ebd3331e7a50ab1163f89c31b75db32c5dbe066b/bin/fliegdoc.js#L19
something like:
```js
const moduleName = 'fliegdoc';
const { config, filepath } = cosmiconfigSync(moduleName, {
	searchPlaces: [
		`.${moduleName}rc.js`,
		`.${moduleName}rc.cjs`,
		`${moduleName}.config.js`,
		`${moduleName}.config.cjs`
	]
}).search();
```
_Originally posted by @fussel178 in https://github.com/fliegwerk/fliegdoc/pull/8#discussion_r554164856_
